sitewit/models.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+from sitewit.services import SitewitService
+
+
+class User(object):
+    def __init__(self, name, email, token=None):
+        self.name = name
+        self.email = email
+        self.token = token
+
+
+class Account(object):
+    _sitewitservice = None
+    DEFAULT_TIME_ZONE = 'Pacific Standard Time'
+
+    def __init__(self, account_data, user_data=None):
+        self.id = account_data.get('accountNumber')
+        self.token = account_data.get('token')
+        self.status = account_data.get('status')
+        self.url = account_data.get('url')
+        self.time_zone = account_data.get('timeZone', self.DEFAULT_TIME_ZONE)
+        self.site_id = account_data.get('clientId')
+        self.password = account_data.get('password')
```

Comment:
We don't set it now.  And it's not present in response  and since our model is wrapper around response, I think we shouldn't have it as attribute.